example-config/run_tests.py

```diff
@@ -1,32 +1,40 @@
 #!/usr/bin/env python2.7
+
+"""
+unit tests for kastenwesen
+"""
+
 import os
 import subprocess
 import logging
 
-script_dir = os.path.dirname(os.path.realpath(__file__))
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
 
 
 def assert_run_okay(cmd):
-    kastenwesen_dir = script_dir + "/../"
-    kastenwesen = kastenwesen_dir + "kastenwesen.py "
-    cmd = kastenwesen + cmd
+    """runs kastenwesen {cmd} and fails, when the command fails"""
+    kastenwesen = os.path.join(SCRIPT_DIR, '../kastenwesen/kastenwesen.py')
+    cmd = '{kw} {cmd}'.format(kw=kastenwesen, cmd=cmd)
```

Comment:
I think it would be better to split out cosmetic changes such as this line into a separate pull request with `git commit -p`.